tests/api.c

```diff
@@ -2236,6 +2262,1080 @@ static int test_wolfSSL_UseOCSPStaplingV2(void)
 
 } /*END test_wolfSSL_UseOCSPStaplingV2*/
 
+/*----------------------------------------------------------------------------*
+ |  Wolfcrypt
+ *----------------------------------------------------------------------------*/
+
+/*
+ * Unit test for the wc_InitMd5()
+ */
+static int test_wc_InitMd5 (void)
+{
+#ifndef NO_MD5
+
+    Md5 md5;
+    int ret, flag;
+
+    printf(testingFmt, "wc_InitMd5()");
+
+    flag = 0;
+
+    /* Test good arg. */
+    ret = wc_InitMd5(&md5);
+    if (ret != 0) {
+        flag = SSL_FATAL_ERROR;
+    }
+
+    /* Test bad arg. */
+    if (!flag) {
+        ret = wc_InitMd5(NULL);
+        if (ret != BAD_FUNC_ARG) {
+            flag = SSL_FATAL_ERROR;
+        }
+    }
+
+    printf(resultFmt, flag == 0 ? passed : failed);
+
+    return 0;
```

Comment:
Since 0 is returned both inside and outside the if/else, I would restructure this to only be a single "#ifndef NO_MD5... #endif", then return 0 outside the #endif for both cases.